src/app/models/indicator-parameter.model.ts

```diff
@@ -0,0 +1,8 @@
+/* tslint:disable:variable-name */
```

Comment:
Do any of these params actually fail the variable name check? It's probably better to only disable checks when necessary, right?